src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/layout.tsx

```diff
@@ -2,24 +2,54 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// import Link from "next/link";
+"use client";
+
+// import Link from "next/image";
+import Image from "next/image";
 import { ReactNode } from "react";
 import { FixNavigation } from "../../../../../../components/client/FixNavigation";
 import styles from "./fix.module.scss";
+import { useRouter } from "next/navigation";
+import imageNavLeft from "./images/nav-arrow-left.svg";
+// import imageNavRight from "./images/nav-arrow-right.svg"
+
 // import { getServerSession } from "next-auth";
 // import { notFound } from "next/navigation";
 
 // import "../../../../client/css/index.css";
 // import { authOptions } from "../../../api/utils/auth";
+
 export type Props = {
+  // navArrowNextVisible: boolean;
+  navArrowBackVisible: boolean;
   children: ReactNode;
 };
 
 // TODO:
 // Add logic to protect routes for specific users (premium/not, scan started/not)
 // Question: Can FXA redirect user back to specific URL (for returning upgrade users during fix data broker)
 
+function NavigationArrowBack() {
+  const router = useRouter();
+
+  return (
+    <button className={styles.navArrowBack} onClick={() => router.back()}>
+      <Image alt="" src={imageNavLeft} />
```

Comment:
nit: might need a localized "Back" alt/aria-label for screen readers.

Reply:
Note: This comment still stands, even if it's not technically going "back" in history, but more to the previous step.